recipes/stir/meta.yaml

```diff
@@ -0,0 +1,53 @@
+
+{% set name = "STIR" %}
+{% set version = "v3_0_1" %}
+{% set sha256 = "4f257fa7a4b15c9d6eddf981d80b79fc27a3ed10df4b8184d507d82a22f3596e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/olivierverdier/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py2k]
+  skip: true  # [win and py<36]
+  number: 0
+
+requirements:
+  build:
+    - python
+    - cmake
+    - boost
+    - swig
+    - toolchain
+  run:
+    - python
+    - numpy >=1.8
+
+test:
+  imports:
+    - stir
+  commands:
```

Comment:
There are a bunch of STIR command line utilities that are also installed, like `FBP2D` and `zoom_image`. You could also try running one of those with `--help` or whatever they support, just to make sure the binaries are linked correctly.

Reply:
Proper tests (which call the installed binaries) are now added. (No `--help` for any of the STIR binaries, unfortunately; plus all commands return an error if not given an input file in argument)